op-e2e/system_test.go

```diff
@@ -658,6 +661,24 @@ func calcGasFees(gasUsed uint64, gasTipCap *big.Int, gasFeeCap *big.Int, baseFee
 	return x.Mul(x, new(big.Int).SetUint64(gasUsed))
 }
 
+// calcL1GasUsed returns the gas used to include the transaction data in
+// the calldata on L1
+func calcL1GasUsed(data []byte, overhead *big.Int) *big.Int {
+	var zeroes, ones uint64
+	for _, byt := range data {
+		if byt == 0 {
+			zeroes++
+		} else {
+			ones++
+		}
+	}
+
+	zeroesGas := zeroes * params.TxDataZeroGas
```

Comment:
these values should be hardcoded for the test.

Reply:
done.